### PR TITLE
Makefile for pieces with comment

### PR DESCRIPTION
**Pull request by fibsifan**
I heavily refactored the makefile for pieces with a comment.

This is concurrent with the changes in nun-singet-und-seid-froh/docker - every step now uses its own docker container if the corresponding program is not installed locally

### CODE REVIEW

**Comment by fibsifan** [comment]
Yet to be done: put the .iltex files on gitignore and let them be cleaned up by `make clean`.

File: include/Makefile-comment [hunk 1]
```diff
@@ -3,49 +3,51 @@ SHELL=/bin/bash
...
 	$(LILY) -dno-point-and-click -o "$(DIRNAME)" $<
-	gs -dBATCH -dNOPAUSE -sDEVICE=pdfwrite -sOutputFile="$(DIRNAME)"-sheet.pdf "$(DIRNAME)".pdf
-	# mv "$(DIRNAME)".pdf "$(DIRNAME)-sheet.pdf"
+	mv "$(DIRNAME).pdf" "$(DIRNAME)-sheet.pdf"
+
+translation.iltex: translation.ilf
+	$(PYTHON) ../../../tools/ilf2iltex.py translation.ilf
 
-$(DIRNAME)-commentary.pdf:
+$(DIRNAME)-commentary.pdf: commentary.tex translation.iltex
```
Comment (jonathan-scholbach) [inline include/Makefile-comment]: Commentaries do not necessarily have a translation. So, I think, this dependency works for Virga Jesse, but might fail for future commentaries, which do not happen to have a transaltion.ilf

Comment (fibsifan) [reply include/Makefile-comment]: fixed via last example from http://kirste.userpage.fu-berlin.de/chemnet/use/info/make/make_4.html#SEC24

**Comment by fibsifan** [comment]
This depends on #36

**Review by jonathan-scholbach — APPROVED** [review]

**Pull request merged**